sdk/python/feast/feature_server.py

```diff
@@ -50,13 +51,22 @@ def get_app(
     registry_ttl_sec: int = DEFAULT_FEATURE_SERVER_REGISTRY_TTL,
 ):
     proto_json.patch()
-
-    app = FastAPI()
     # Asynchronously refresh registry, notifying shutdown and canceling the active timer if the app is shutting down
     registry_proto = None
     shutting_down = False
     active_timer: Optional[threading.Timer] = None
 
+    @asynccontextmanager
+    async def lifespan(app: FastAPI):
+        nonlocal shutting_down
+
```

Comment:
Can we also move `async_refresh` call here? Seems like more logical place for it

Reply:
@tokoko Sure, I changed async_refresh and the original shutdown_event into lifespan.